binder/environment.yml

```diff
@@ -40,3 +40,7 @@ dependencies:
   - ipykernel
   - pytest
   - pytest-xdist
+  # this is needed for PyDDA performance
+  - jaxopt
+  - tensorflow>=2.6
+  - tensorflow-probability>=0.24
```

Comment:
I've pinned this use the currently latest available version on conda-forge. Otherwise it resolved to a package from anaconda pkgs/main which probably broke the process in #71.